code/__HELPERS/game.dm

```diff
@@ -471,3 +472,12 @@ proc/pollCandidates(var/Question, var/be_special_type, var/antag_age_check = 0,
 			candidates.Remove(G)
 
 	return candidates
+
+/proc/window_flash(client/C)
+	if(ismob(C))
+		var/mob/M = C
+		if(M.client)
+			C = M.client
+	if(!C.prefs.windowflashing || !C)

```

Comment:
oh, this should be the other way around, or it'll evaluate `C.prefs.windowflashing` when `C` is null and runtime anyways